plugins/traction_innkeeper/traction_innkeeper/v1_0/innkeeper/routes.py

```diff
@@ -917,6 +919,41 @@ async def innkeeper_authentications_api_delete(request: web.BaseRequest):
     return web.json_response({"success": result})
 
 
+@docs(tags=[SWAGGER_CATEGORY], summary="Fetch the server configuration")
+@response_schema(AdminConfigSchema(), 200, description="")
+@innkeeper_only
+@error_handler
+async def innkeeper_config_handler(request: web.BaseRequest):
+    context: AdminRequestContext = request["context"]
+    # use base/root profile for server config, use Tenant Manager profile
+    # this is to not get the Innkeeper tenant's config, but the server cfg
+    mgr = context.inject(TenantManager)
+    profile = mgr.profile
+
+    config = {
+        k: (
```

Comment:
This is nice, but I like things to be more readable than shorter. I'm assuming that `k` = `key`. Imho is better just to write key here so other people don't need to decipher it.
It's not a big deal or anything but saving a couple characters doesn't seem worth it.

Reply:
Good idea